app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert len(set(split_integer(8, 4))) == 1
```

Comment:
Sometimes it is preferable to make an explicit comparison instead of additional modifications over data to improve the readability
```suggestion
    assert split_integer(8, 4) == [2, 2, 2, 2]
```